test/fixtures/test_grammars/next_sibling_from_zwt/corpus.txt

```diff
@@ -0,0 +1,2 @@
+(source_file 
```

Comment:
This corpus file doesn't look like it's formatted right. There's supposed to be a source document followed by a separator line like `-----`, and then the expected parse tree.

Reply:
My bad, completely blanked on finishing this when I wrote up the initial tests.